src/app/models/form.server.model.ts

```diff
@@ -501,6 +495,32 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     return form.save()
   }
 
+  FormSchema.statics.getDashboardForms = async function (
+    this: IFormModel,
+    userId: IUserSchema['_id'],
+    userEmail: IUserSchema['email'],
+  ): Promise<DashboardFormView[]> {
+    return (
+      this.find()
+        // List forms when either the user is an admin or collaborator.
+        .or([{ 'permissionList.email': userEmail }, { admin: userId }])
+        // Filter out archived forms.
+        .where('status')
+        .ne(Status.Archived)
+        // Project selected fields.
+        .select('_id title admin lastModified status form_fields')
```

Comment:
might be cleaner to extract this into a constants file somewhere

Reply:
Only used here, no need to extract for now